src/ai/backend/manager/models/session.py

```diff
@@ -166,6 +168,8 @@ class SessionStatus(enum.Enum):
     )
 )
 
+PRIVATE_SESSION_TYPES = (SessionTypes.SYSTEM,)
```

Comment:
As a follow-up work, this should be refactored to handle "special type" sessions like SFTP containers which allow direct access without AppProxy.